libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -2435,22 +2435,90 @@ async def ainvoke(
 
 
 class RunnableBinding(RunnableSerializable[Input, Output]):
-    """
-    A runnable that delegates calls to another runnable with a set of kwargs.
+    """Wrap a runnable with additional functionality.
+
+    A RunnableBinding can be thought of as a "runnable decorator" that
+    preserves the essential features of Runnable; i.e., batching, streaming,
+    and async support, while adding additional functionality.
+
+    Any class that inherits from Runnable can be bound to a `RunnableBinding`.
+
+    Runnables expose a standard set of methods for creating `RunnableBindings`
+    or sub-classes of `RunnableBindings` (e.g., `RunnableRetry`,
+    `RunnableWithFallbacks`) that add additional functionality.
+
+    These methods include:
+
+    - `bind`: Bind kwargs to pass to the underlying runnable when running it.
+    - `with_config`: Bind config to pass to the underlying runnable when running it.
+    - `with_listeners`:  Bind lifecycle listeners to the underlying runnable.
+    - `with_types`: Override the input and output types of the underlying runnable.
+    - `with_retry`: Bind a retry policy to the underlying runnable.
+    - `with_fallbacks`: Bind a fallback policy to the underlying runnable.
+
+
+    Example:
+
+    `bind`: Bind kwargs to pass to the underlying runnable when running it.
+
+        .. code-block:: python
+
+            # Create a runnable binding that invokes the ChatModel with the
+            # additional kwarg `stop=['-']` when running it.
+            from langchain.chat_models import ChatOpenAI
+
+            model = ChatOpenAI()
+            model.invoke('Say "Parrot-MAGIC"', stop=['-']) # Should return `Parrot`
+
+            # Using it the easy way via `bind` method which returns a new
+            # RunnableBinding
+            runnable_binding = model.bind(stop=['-'])
+            runnable_binding.invoke('Say "Parrot-MAGIC"') # Should return `Parrot`
+
+
+        Can also be done by instantiating a RunnableBinding directly (not recommended):
+
+        .. code-block:: python
+
+            from langchain.schema.runnable import RunnableBinding
+
+            runnable_binding = RunnableBinding(
+                bound=model,
+                kwargs={'stop': ['-']} # <-- Note the additional kwargs
+            )
+
+            runnable_binding.invoke('Say "Parrot-MAGIC"') # Should return `Parrot`
     """
 
     bound: Runnable[Input, Output]
```

Comment:
why not `runable`? `bound` name is confusing. Before reading docstring, I thought it is "bound/not bound" bool .

Reply:
`bound` is technically correct, but yeah maybe runnable would've been less confusing. @nfcampos :eye: 

naming is hard :)